### DISCUSSION TITLE
Code review ІМ-11

### DISCUSSION THREAD

**Discussion by Nester44** [HowProgrammingWorks/LiveQA#76] [Code review]
Пишу браузерную игру "Doodle Jump":
Задача прыгать по платформам и набрать как можно больше счёт. В будущем планирую добавить уровни сложностей, разные виды платформ и NPC. Буду благодарен за ревью [проекта](https://github.com/Nester44/Doodle-Jump)

**Comment by tshemsedinov** [comment]
See review: https://github.com/Nester44/Doodle-Jump/issues/1
